src/helpers/schema.sql

```diff
@@ -4,5 +4,6 @@ CREATE TABLE subscribers (
   created BIGINT NOT NULL,
   verified BIGINT NOT NULL DEFAULT 0,
   PRIMARY KEY (email, address),
+  UNIQUE KEY idx_address_email (address, email),
```

Comment:
it's redundant indexing. You can choose whether `PRIMARY` or `UNIQUE` key.

Reply:
You're correct. As the index was already added, let's leave it, it's still useful, as we still need an index on `address`

Reply:
I assume with the primary key you already get an index on both fields so its maybe useless

Reply:
Composite index only add the index on the first column, and not subsequent one. So when querying only by `address`, the PRIMARY KEY can not be used (try running `explain select * from subscribers where address= '';`)

We can replace that index by a simple `KEY address (address)`

Reply:
Yes, it does, as long as address is the first column